.github/labeler.yml

```diff
@@ -0,0 +1,14 @@
+area/csi:
+- csi/**/*
+area/ci:
+- .travis
+- .github/**/*
+area/osd-cmd:
+- osd/**/*
+area/api-proto:
+- api/api.pb.go
+- api/api.proto
+area/api:
+- api/*
```

Comment:
Let's start with a wider area and keep everything under `api/*` as `area/api`.

Reply:
Ok, I kept api/server/sdk as well since I think it's helpful to know about SDK server changes. The proto regex I've removed.

Reply:
I think these labels will be even more useful in porx